### PR TITLE
Fixes some inventory stuff

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

When taking off a jumpsuit, items will no longer fall off of you if you're able to equip them in their current slot without said jumpsuit.

# Changelog

:cl:  
bugfix: fixes items falling off when removing a jumpsuit when you're still able to equip them in their current slot without one
/:cl:

### CODE REVIEW

**Comment by MortimerTMorose** [comment]
this is the single best PR I've ever seen

File: code/modules/mob/living/carbon/inventory.dm [hunk 1]
```diff
@@ -116,6 +116,9 @@
 		legcuffed = null
 		if(!QDELETED(src))
 			update_inv_legcuffed()
+	else if(I == internal)
+		if(!QDELETED(src))
+			update_internals() // no more "bluespace" internals tanks
```
Comment (ReddicusDragon) [inline code/modules/mob/living/carbon/inventory.dm]:
```suggestion
```
As of #18301, tanks will now close if they are dropped

**Review by ReddicusDragon — DISMISSED** [review]
See above comment

**Comment by iamthesilly** [comment]
fixed

**Review by Moltijoe — CHANGES_REQUESTED** [review]
please update changelog

**Comment by iamthesilly** [comment]
> please update changelog

Done

**Review by Moltijoe — APPROVED** [review]

**Comment by iamthesilly** [comment]
@ReddicusDragon i fixed the thing you asked me to fix

**Pull request merged**